src/page/main/main.tsx

```diff
@@ -0,0 +1,141 @@
+import React, {Props} from 'react';
+import {
+  View,
+  Text,
+  TextInput,
+  TouchableHighlight,
+  ScrollView,
+} from 'react-native';
+import {styles} from './mainCss';
+import {FontAwesomeIcon} from '@fortawesome/react-native-fontawesome';
+import {
+  faBars,
+  faShoppingCart,
+  faSearch,
+} from '@fortawesome/free-solid-svg-icons';
+// import {} from '@fortawesome/free-brands-svg-icons';
+import Logo from '../../resource/images/logo.svg';
+
+const CircleIcon: React.FC = (Props: Porps) => {
```

Comment:
Please never create several components in one file. Create `components` folder in root folder and put re-usable components there